src/index.ts

```diff
@@ -1,93 +1,154 @@
-import * as core from '@actions/core';
-import * as github from '@actions/github'
-import { LinearClient } from '@linear/sdk'
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import { LinearClient } from "@linear/sdk";
 import { isNil } from "lodash";
 
 const PR_TITLE_UPDATE_KEYWORD = "x";
 
 type PullRequest = {
-    number: number;
-    title: string;
-    head: {
-        ref: string;
-        repo: {
-            name: string;
-            owner: {
-                login: string;
-            }
-        } | null,
-    },
-}
+  number: number;
+  title: string;
+  body: string | null;
+  head: {
+    ref: string;
+    repo: {
+      name: string;
+      owner: {
+        login: string;
+      };
+    } | null;
+  };
+};
 
-export const getIssueIdFromBranchName = (branchName: string): string | null => {
-    const match = branchName.match(/^feature\/([a-z]+\-\d+)\-/);
-    if (isNil(match)) return null;
-    const branchStoryId = match[1];
-    return branchStoryId.toUpperCase();
-}
+export const getLinearIssueIds = (pullRequest: PullRequest) => {
+  const issueIds: string[] = [];
+
+  // From the branch name
+  const branchName = pullRequest.head.ref;
+  let match = branchName.match(/^([a-z]+\-\d+)\-/);
```

Comment:
I've removed `feature` in Linear's configuration. It's now only `identifier-title`.

Reply:
Wouldn't that be much more easily mistaken? Something like `add-10-new-pixels` would match. Since we already have `release/` and `hotfix/` I think it made sense to have `feature/`.

Reply:
I’ve removed it because we already had it removed in Shortcut.

Also seems like boilerplate. In Shortcut, we at least had feature/ & bug/ but bug is just a label in Linear, so everything’s a feature.